release/RELEASE.md

```diff
@@ -31,8 +31,12 @@ patches we backport to the 1.0 release branch).
    $ ./release.sh snapshot cc880e2 0.1.2
    cc880e290b2311d0bf05d58c7d75c50784c0131c 0.1.2-snapshot.20200513.4174.0.cc880e29
    ```
-   Then open a PR with the changed `LATEST` file, and add the `Standard-Change`
-   label.
+   Then open a PR _to be merged in `main`_ (even if it's for a maintenance release)
+   with the changed `LATEST` file, add the line produced by the `release.sh`
+   invocation in a meaningful position and add the `Standard-Change` label. It
```

Comment:
The extra warning here isn’t actually that important. You can rerun only that check and it finishes in less than a minute.

Reply:
still nice to add it initially though so you can have mergify take over, so no objection to add it.

Reply:
I'd leave it because AZP forces you to wait for all the other checks to complete before re-running it, which is one more annoying manual-step-after-waiting.